_std/defines/component_defines.dm

```diff
@@ -144,3 +149,10 @@
 // obj/critter signals
 // When an obj/critter dies
 #define COMSIG_OBJ_CRITTER_DEATH "obj_critter_death"
+
+/// general return values //
+#define RETURN_SUCCESS 0x001
+#define RETURN_FAILURE 0x002
```

Comment:
Are these supposed to be general purpose flags for returning from signals? If so I don't think that's a good idea. I'd rather have each signal define its own, especially considering how these likely clash with already existing return flags. (Also it's a bit weird how you can have success and failure at the same time. I'd expect success to be one flag and the absence of success deemed as a failure.)

Reply:
on second though I wasn't even using those anyways